fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -1143,7 +1143,7 @@ namespace fileBackup {
 
 							Void _ = wait(taskBucket->keepRunning(tr, task)
```

Comment:
Just a random thought: 
If the return value of wait is Void, we no longer needs to assign it to Void _.

Reply:
We do on release-6.0 still, as the actor compiler change is only on master.  (It also changed `Void _ =` into an error, so this wouldn't build if it was supported.)